docs/topics/genericgateway.md

```diff
@@ -47,18 +64,60 @@ When calling the Generic Gateway `simpleevent` endpoint, you must include a cust
 }
 ```
 
-`Ref` and `commit` values are used to configure the specific revision that Brigade will pull from your repository, if you're using one. If both values are missing, then your Build's `ref` will be set to `master`. If you do not wish to provide any payload, just include an empty JSON object (like `{}`).
+`ref` and `commit` values are used to configure the specific revision that Brigade will pull from your repository, if you're using one. If both values are missing, your Build's `ref` will be set to `master`. Of course, you can also provide any other values you need.
+If you do not wish to provide any payload, just send an empty JSON object (`{}`). 
+
+### Calling the CloudEvent endpoint
```

Comment:
Could you please include a simple `curl` POST command with the payload (both for this and for a simple event?)
It would be extremely helpful to try it out.

Reply:
great idea, adding